### DISCUSSION TITLE
CUSTOM ASCII LOGO

### DISCUSSION THREAD

**Discussion by nolbertocastro** [o2sh/onefetch#970] [Q&A]
Is there a way in which I san change the default ASCII logo, for a custom one? and how can I change the color of the logo?

**Comment by spenserblack** [comment]
`--ascii-input` for custom ASCII and `-c`/`--ascii-colors` for custom colors. Note that the first color is the primary color, used for the labels.

```bash
onefetch --ascii-input '{0}black{1}red{2}green' -c 0 1 2
```

**Reply by o2sh** [reply]
Adding to @spenserblack's comment, you can check [this page](https://github.com/o2sh/onefetch/wiki/command-line-options) for more customization options - equivalent to running `onefetch --help`.
